### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ayalfishey**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,7 +9,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".AddActivity"></activity>
```
Comment (HannahShulman) [inline app/src/main/AndroidManifest.xml]: Haveing verbose names helps... you may want to call this AddTodoActivity

File: app/src/main/java/com/ayal/todo/DataManager.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.ayal.todo;
+
+import java.util.ArrayList;
+
+public class DataManager {
+    private static ArrayList<Todo> todos = new ArrayList<>();
+
+    public static void AddToList(Todo task){
```
Comment (HannahShulman) [inline app/src/main/java/com/ayal/todo/DataManager.java]: method names always should start with lower case

File: app/src/main/java/com/ayal/todo/Todo.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        this.todo = todo;
+    }
+
+    public String getTodo() {
+        return todo;
+    }
+
+    public void setTodo(String todo) {
+        this.todo = todo;
+    }
```
Comment (HannahShulman) [inline app/src/main/java/com/ayal/todo/Todo.java:16]: Nice and clean :)

File: app/src/main/java/com/ayal/todo/DataManager.java (hunk 2 above)
Comment (ayalfishey) [reply app/src/main/java/com/ayal/todo/DataManager.java]: opps

**Review by HannahShulman — CHANGES_REQUESTED** [review]
Was nice reviewing your code. You are doing really well. I enjoy you being part of the course.
Please see my comments, and re-request review when done. Thanks, and just continue as is.

**Pull request merged**